src/index.html

```diff
@@ -5,9 +5,57 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div class="product" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="product__photo"
+      >
+
+      <h3 class="product__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="product__code">
+        Product code: 195434
+      </p>
+
+      <div class="product__review">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product__reviews">
+          Reviews: 5
+        </p>
+      </div>
+
+      <div class="product__price">
+        <p class="product__priceText">
+          Price:
+        </p>
+        <p class="product__priceDollar">
```

Comment:
```suggestion
        <p class="product__price-dollar">
```
Use one naming convention for all classes